guides/user/kubernetes-v2/deploy-manifest/index.md

```diff
@@ -0,0 +1,173 @@
+V2](/setup/install/providers/kubernetes-v2).
+
+There are two things to cover here:
+
+* [How to specify which manifest to deploy](#specifying-your-manifest)
+* [How to override artifacts in the manifest](#overriding-artifacts)
+
+At a high level, you specify a manifest to deploy, and optionally, can override
+certain artifacts (as fields) at runtime (e.g. the docker image to use).
+
+## Specifying Your Manifest
+
+Depending on your needs, there is more than one way to specify the manifest
+that you want to deploy:
+
+* [Statically: directly in the pipeline](#statically-specified-manifests)
+* [Dynamically: bound at runtime using an
+  artifact](#dynamically-specified-manifests)
+
+In either case, we want to start by selecting the __Deploy (Manifest)__ stage
+from the stage selector as shown here:
+
+{%
+  include
+  figure
+  image_path="./deploy-manifest.png"
+%}
+
+> :warning: The regular __Deploy__ stage is used to deploy more opinionated
+> "Server Groups" in Spinnaker, using either another provider, or the
+> Kubernetes V1 provider.
+
+### Statically specified manifests
+
+If you know ahead of time what you expect to deploy a certain manifest (even if
+you don't know what version of your docker image it will run) you can declare
+it directly in the pipeline the manifest specification.
+
+{%
+  include
+  figure
+  image_path="./in-pipeline.png"
+  caption="Notice that by selecting \"Text\" as the __Manifest Source__, we get
+  to enter the Manifest YAML by hand."
+%}
+
+Of course, if you are generating your pipeline definitions rather than entering
+them into the UI, the stage definition would look more like:
+
+```json
+{
+  "name": "Deploy my manifest",   // human-readable name
+  "type": "deployManifest",       // tells orchestration engine what to run
+  "account": "nudge",             // account (k8s cluster) to deploy to
+  "cloudProvider": "kubernetes",
+  "source": "text",
+  "manifest": {
+                                  // manifest contents go here
+  },
+  "moniker": {                    // used to specify app & cluster for grouping
+                                  // resources in UI
+    "app": "xnat",
+    "cluster": "c7",
+  }
+}
+```
+
+### Dynamically specified manifests
+
+If you are storing your manifests outside of Spinnaker's pipeline repository,
+or want a single deploy stage to be able to deploy a variety of manifests, you
+can specify your manifest using an [Artifact](/reference/artifacts).
+
+The idea is: artifacts in Spinnaker allow you to reference remote, deployable
+resources. In our case, artifacts will have to reference a text file
+(containing a Manifest specification) stored in GitHub, or an object store
+(like GCS).
+
+There are already some guides describing how to trigger off of changes to these
+artifacts, and supply them to pipelines. See:
+
+* [Consuming GitHub Artifacts](/guides/user/artifacts/github)
+* [Consuming GCS Artifacts](/guides/user/artifacts/gcs)
```

Comment:
Saying "There are already some guides..." colors this document as new, which is true but won't always be true. It's more mundane to say "For more information about triggering based on changes...," but it works better. (Yeah, I also changed "off of" to "based on" cause "off of" is hard-ish to read.